src/fragments/lib/graphqlapi/js/optimistic-ui.mdx

```diff
@@ -328,44 +328,45 @@ const updateMutation = useMutation({
   mutationFn: async (
     realEstatePropertyDetails: UpdateRealEstatePropertyInput
   ) => {
-    const updatedRealEstateProperty = await API.graphql<
+    const response = await API.graphql<
       GraphQLQuery<UpdateRealEstatePropertyMutation>
     >({
       query: mutations.updateRealEstateProperty,
       variables: { input: realEstatePropertyDetails },
     });
 
-    return updatedRealEstateProperty;
+    return response?.data?.updateRealEstateProperty;
   },
   // When mutate is called:
-  onMutate: async (
-    newRealEstateProperty: UpdateRealEstatePropertyInput
-  ) => {
-  // Cancel any outgoing refetches
-  // (so they don't overwrite our optimistic update)
-  await queryClient.cancelQueries({
-    queryKey: ["realEstateProperties", newRealEstateProperty.id],
-  });
+  onMutate: async (newRealEstateProperty) => {
+    // Cancel any outgoing refetches
+    // (so they don't overwrite our optimistic update)
+    await queryClient.cancelQueries({
+      queryKey: ["realEstateProperties", newRealEstateProperty.id],
+    });
+
+    await queryClient.cancelQueries({
```

Comment:
is this case covered in the 344?

Reply:
Sorry, can you clarify your question?

Reply:
Both cancellations here are required - one invalidates the cache for the list view, the other invalidates the cache for the detail view.